packages/react-examples/.storybook/preview-loader.js

```diff
@@ -1,11 +1,15 @@
 // @ts-check
-import * as fs from 'fs';
+import * as fs from 'fs-extra';
 import * as path from 'path';
 import * as webpack from 'webpack';
 
 /**
  * Replace the PACKAGE_NAME token in preview.js with the actual current package name
  * (or error if the package doesn't have examples).
+ *
+ * Also if this is the storybook for `@fluentui/react`, replace the REACT_DEPS token
```

Comment:
This is the interesting part for making FUIR storybook show re-exported stuff :)